### PR TITLE
Bakcups and recovery

### PR DESCRIPTION
**Pull request by LotteKh**
## Purpose

Make clearer how to make backups in DTW and MW and the differences between the two.

## Changes

_Restructured and edited Account Recovery
_Edited Make backup... in the Desktop Wallet
_Rewrote Deciding between DTW and MW
_Edited  Exporting and Importing in MW

### CODE REVIEW

File: source/shared/net/mobile-wallet/export-import-mw.rst [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
+Make a backup of identities and accounts in the Mobile Wallet
+=============================================================
 
 .. contents::
    :local:
    :backlinks: none
 
-We recommend that you backup your identities and accounts by exporting them from the Mobile Wallet. This way,
-all your accounts, identities, and your address book are saved in a file you can store in a safe location.
+To make sure that you have a backup of your accounts, identities, and addresses, we strongly recommend that you export the data from the Mobile Wallet to a file. This way, all your accounts, identities, and your address book are saved in a file you can store in a safe location.
```
Comment (concordium-cl) [inline source/shared/net/mobile-wallet/export-import-mw.rst:11]: I'd suggest to make it a bit clearer here as well. we recommend that you export the data and then move the export file to a location off your phone. I don't think the last sentence is strong enough to cover it.

**Review by concordium-cl — CHANGES_REQUESTED** [review]

**Review by concordium-cl — APPROVED** [review]

**Comment by jens-concordium** [comment]
Oh, I was too late! I see it has already been merged now.

File: source/shared/net/concepts/id-accounts.rst [hunk 2]
```diff
@@ -46,7 +46,7 @@ a number of public and private keys, a signature from the identity provider, as
 well as a number of secret values the user must use to be able to use the
 identity to create accounts.
 
-You can create identities in the :ref:`Desktop Wallet <create-initial-account-desktop>` or in the :ref:`Mobile Wallet <mobile-get-started>`. The |Net| release presently supports the Notabene identity issuance flow.
+You can create identities in the :ref:`Desktop Wallet <create-initial-account-desktop>` or in the :ref:`Make a backup of identities and accounts in the Mobile Wallet <export-import-mw>`. The |Net| release presently supports the Notabene identity issuance flow.
```
[the remark this answers is not in the document]
Comment (LotteKh) [reply source/shared/net/concepts/id-accounts.rst]: Has been changed

**Comment by LotteKh** [comment]
I'll make the changes

**Pull request merged**